wiki/intro/faq/de.yml

```
@@ -0,0 +1,34 @@
+title: "FAQ"
+description: "Häufig gestellte Fragen"
+
+q-info: "Was macht diese Mod?"
+a-info: "Diese Mod ermöglicht es dir, in JavaScript erstellte Skripte zu verwenden, um deinen Server zu verwalten, neue Blöcke und Items hinzuzufügen, Rezepte zu ändern, eigene Event-Handler für Quest-Mods hinzuzufügen und vieles mehr!"
+
+q-how-use: "Wie benutze ich sie?"
+a-how-use: "Wenn du das Spiel zum ersten Mal mit der Mod startest, wird im Minecraft-Verzeichnis ein Ordner namens `kubejs` erstellt, der Beispiele und eine README.txt enthält. Schau einfach mal rein!"
+
+q-js-help: "Aber ich kenne mich überhaupt nicht mit JavaScript aus!"
+a-js-help: "Das ist okay, es gibt hier genug Beispiele und vorgefertigte Skripte. Und wenn du trotzdem nicht weiterkommst, kannst du im Support-Kanal unseres Discords nach Hilfe fragen, aber bitte sei bitte möglichst genau."
```

The German version here sounds much more polite than the English version but I am fine with that.
Either way, there should be a period after "Das ist okay" and the next sentence should then start with a capital letter. Would also replace "genug" with "genügend".
There is a duplicate "bitte" at the end of the sentence.

```suggestion
a-js-help: "Das ist okay. Es gibt hier genügend Beispiele und vorgefertigte Skripte. Und wenn du trotzdem nicht weiterkommst, kannst du im Support-Kanal unseres Discords nach Hilfe fragen, aber bitte sei möglichst genau."
```